### ISSUE TITLE
Property data type in "@var" comments isn't auto-fixed

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#682]
I know, that for function arguments and return value we're automatically fixing data types (e.g. "bool" to "boolean", etc.).

However I've noticed, that same doesn't happen for class property data types:
```php
<?php

class Sample {

	/**
	 * Don't execute hooks, before event processing
	 *
	 * @var bool
	 */
	public $SkipBeforeHooks = false;

}
```

**Comment by aik099** [comment]
Might be fixed in #1061.

**Comment by gsherwood** [comment]
> Might be fixed in #1061.

Confirmed fixed.
